lib/ansible/utils/module_docs_fragments/junos.py

```diff
@@ -24,6 +24,11 @@ class ModuleDocFragment(object):
 options:
   provider:
     description:
+      - B(Deprecated)
+      - "Starting with Ansible 2.5 we recommend using C(connection: network_cli) or C(connection: netconf)."
+      - This option is only required if you are using eAPI.
```

Comment:
@gundalow `eAPI` shouldn't be in `junos`.

Reply:
Good spot, fixed